### DISCUSSION TITLE
Flashing update loose settings?

### DISCUSSION THREAD

**Discussion by DeNohvo** [Jyers/Marlin#122] [Q&A] [answered]
Will flashing updates reset the printer?

What about moving from 3x3 to 5x5 bin files. Assume that means i need to run the mesh bed level?

**Comment by t3chguy** [comment, marked as answer]
Whenever you update Marlin in general, not just this fork, it is recommended you reset to defaults as values in EEPROM may have changed meaning.

**Reply by DeNohvo** [reply]
I was afraid this would be the answer, I've been struggling with allot of first layer problems. Finally have my settings dialed in but wanted to flash a newer update.

However the first layer problems have been mostly mechanical rather than software, requiring me to redo allot of the factory assessbly.

**Reply by darco415** [reply]
you can always reflash then run a test print...adjust accordingly

**Reply by Woodcat64** [reply]
If you just worry about motion/step ratios and offsets. Write those down. Load new FW and later put your values back in.

**Reply by DeNohvo** [reply]
Yep but the one time i didn't write it down after adjusting z axis
